docs/platforms/apple/common/enriching-events/screenshots/index.mdx

```diff
@@ -14,6 +14,10 @@ When a user experiences an error, an exception or a crash, Sentry provides the a
 
 This feature only applies to SDKs with a user interface, such as the ones for mobile and desktop applications. In some environments like native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. Another example where a screenshot might not be available is when the event happens before the screen starts to load. So inherently, this feature is a best effort solution.
 
+<Note>
+App hang events will not have a screenshot because the main thread is blocked and we can't interact with UI elements in a background view.
```

Comment:
```suggestion
App hang events will not have a screenshot because the main thread is blocked and Sentry can't interact with UI elements in a background view.
```